### ISSUE TITLE
Error 0x223 on start paused in air on approach

### ISSUE THREAD

**Issue by eaglerainbow** [it0uchpods/IDG-A32X#84]
### Environment
* Flightgear 2018.3.2
* Revision 4781
* Windows 10

### Expected Behavior
It is possible to start the plane in air for practicing landings.

### Current Behavior
Error code: 0x223 raised
Preposterous configuration detected for initialization. Check your position or scenery.

(Note that scenary works just fine - works just fine, if I take the C172P)

### Steps to Reproduce
Start FG Launcher such that the following command line options are generated:

```
--launcher --download-dir=D:/spiele/FlightGear 2018.3.2/data 
--prop:/nasal/local_weather/enabled=false --metar=XXXX 012345Z 15003KT 19SM FEW072 FEW350 25/07 Q1028 NOSIG 
--prop:/environment/weather-scenario=Core high pressure region 
--timeofday=noon --prop:/sim/rendering/multi-sample-buffers=1 
--prop:/sim/rendering/multi-samples=2 --disable-rembrandt 
--prop:/sim/rendering/shaders/skydome=true --enable-fullscreen 
--disable-terrasync --enable-auto-coordination --enable-freeze 
--aircraft-dir=D:/spiele/FlightGear 2018.3.2/data/Aircraft/IDG-A32X 
--aircraft=A320-200-IAE --airport=EDDP --offset-distance=15 
--on-ground=false --vc=250 --altitude=8000 --httpd=8080 
--prop:/sim/rendering/multithreading-mode=CullDrawThreadPerContext 
```

In a nutshell:
Airport EDDP, active runway, 15nm from threshold, 250kts IAS, FL80, NAV1 tuned

Same also happens with 14nm from threshold.

### Additional

I found #73 which seems somewhat similar, but not really my case.  I tried to make a screenshot in Helicopter view, but the attempt to switch the view fails (FGFS is hanging then).

I would be willing to help out with additional logging data, in case you can't reproduce it - I would then need a small tutorial how to enable that then (sorry for that n00b statement).

**Comment by eaglerainbow** [comment]
Scenery file affected: e010n50
I am using the scenery file with SHA1
```
f31b436173e9a29b1e1b535003abe42b7a11ca06 *e010n50.zip
```
right now (filesize: 1236041354 bytes).

**Comment by it0uchpods** [comment]
No, we will never allow this because its not possible. The Aircraft cannot just be "initialized" in the air. The sim is too realistic for that. Perhaps at V1.0 we will make a scenario loadable for this purpose, but the aircraft is changing drastically with many updates, so it won't be done now.

(Hence why the following is said in the error:

**Comment by it0uchpods** [comment]
Sorry for the disappointment.If you wish to practice landings, you can keep going around. This is probably faster than restarting FG each time too.

J

**Comment by eaglerainbow** [comment]
Okay, accepted.
But the error message could be better: Something like "Start up in plain air not supported" (or so)...

**Comment by it0uchpods** [comment]
It is difficult to tell sometimes, when the scenery is missing, as when FG loads, sometimes the values make no sense until the sim finishes loading. I will likely post a frequently asked questions somewhere, that explains all the codes. :)

If I do find a way to do this without messy code, I will!

Kind Regards,
Josh

**Comment by eaglerainbow** [comment]
Okay, not nice, but an FAQ already would help (and if it was just a `FAQ.md` here).
Is the issue with the flaps not working with an axis input with another FAQ entry or better a bug?

**Comment by it0uchpods** [comment]
> Okay, not nice
I don't believe I said anything not nice...? Could you point it out please? I'll get to a FAQ once time allows.

Flap Axis, that is a good question. I will have to check that, as we use a custom flaps handling system. I'll think about it some.

Josh

**Comment by eaglerainbow** [comment]
> I don't believe I said anything not nice...?

No worries - and no offense...
I meant it in the sense of "not nice that it's tricky for you detecting such a situation" (because you don't have consistent scenery data in place). 

> Flap Axis, that is a good question. I will have to check that...

Do you want to have separate issue reported for that from me or would this be more of a burden for you?

**Comment by it0uchpods** [comment]
Oh, I understand now! Thanks for clearing it up, none taken. :)

Yes please, go for it, probably a good idea so I don't forget :)

**Comment by eaglerainbow** [comment]
> Yes please, go for it, probably a good idea so I don't forget :)

Here we go: #85.
However, I am currently asking myself whether this isn't more a feature request (which you asked not to be reported via GH issue tracking)...

**Comment by it0uchpods** [comment]
It is kind of in between, I guess. It should work since flap axis is part of FG. That's close enough for me.

Kind Regards,
Josh
